src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -170,8 +170,12 @@ def update_issue_impact_labels(testcase, issue):
       new_impact = data_types.SecurityImpact.STABLE
     elif testcase.impact_beta_version:
       new_impact = data_types.SecurityImpact.BETA
-    else:
+    elif testcase.is_crash():
       new_impact = data_types.SecurityImpact.HEAD
+    else:
+      # Testcase is unreproducible and does not impact stable and beta branches.
+      # In this case, there is no impact information.
```

Comment:
Should we apply `Security_Impact-None` in such case?

Reply:
No that is when it is legit crash, that does not impact production (e.g. experimental feature). This None impact has always been confusing.

Reply:
Even though we define data_types.SecurityImpact.NONE, we never use/apply it in code. i am even thinking to remove it to avoid confusion ?

Reply:
I take the last comment back, which is we need to define it in code to detect current labels and change as needed.

Reply:
Got it, thanks for clarifying.